NextCloud.py

```diff
@@ -713,6 +714,60 @@ def resend_welcome_mail(self, uid):
         return self.requester.post(url)
 
 
+class UserLDAP(WithRequester):
```

Comment:
It would be great to provide methods for configuration s.a. `edit_ldap_password` and `get_ldap_password` instead of the `edit_ldap_config` accepting a opcode argument. It is possible to go the manual way of reading the documentation and creating those methods by hand, or if they are very similar, it may be worth considering to define a list of e.g. `(name_stem, argname, docstring)` tuples in the module, and creating those methods dynamically in the class constructor using `setattr`.
Think about that, it is not a requirement for this PR to be merged, it can be added later.

Reply:
Great idea, thanks! I've added it as a `TODO` comment below to refactor it like this later.